graph/pull.go

```diff
@@ -28,10 +18,24 @@ type ImagePullConfig struct {
 	OutStream   io.Writer
 }
 
+type Puller interface {
+	Pull(image, name, tag string, endpoint registry.APIEndpoint, imagePullConfig *ImagePullConfig, sf *streamformatter.StreamFormatter) (fallback bool, err error)
```

Comment:
Could you document these parameters?  streamFormatter and fallBack are not immediately obvious.